faq.md

```diff
@@ -33,27 +28,27 @@ No. Do not post links or downloads to such resources or you will be banned.
 
 * Windows 8 or later
 * 64-bit x86 processor supporting AVX
-* A Vulkan compatible GPU from [this list](http://vulkan.gpuinfo.org/) (NVIDIA recommended)
+* A Vulkan (or D3D12) compatible GPU from [this list](https://vulkan.gpuinfo.org/)
 * An XInput-compatible controller
 
 Xenia will check for the minimum supported CPU and GPU on startup and error out
-if one is not detected. Check that your GPU is on [this list](http://vulkan.gpuinfo.org/)
+if one is not detected. Check that your GPU is on [this list](https://vulkan.gpuinfo.org/)
 and that you have the latest drivers installed.
 **You cannot just remove the checks in the code and assume things will work.**
 
 ### What kind of GPU do I need?
 
-Vulkan support and drivers are required. You can check if your GPU is supported by referencing [this list](http://vulkan.gpuinfo.org/).
+Vulkan support and drivers are required. You can check if your GPU is supported by referencing [this list](https://vulkan.gpuinfo.org/).
 
 To get full speed and compatibility the project adapted to Vulkan.
 OpenGL is no longer supported in this project and deprecated.
 
-### Can you add support for Windows Me? How about my Pentium II?
+### Can you add support for Windows 7? How about my Pentium II?
```

Comment:
Unnecessary to be changed.

Reply:
Windows 7 seems more relevant since people ask about it.

Reply:
The question is tongue-in-cheek, it's meant to show the audacity of the question (and Win7 isn't completely out of the question-- those are).